devsetup/scripts/bmaas/network-setup.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# set -x
+
```

Comment:
how about we make this a generic network-setup.sh script for devsetup. right now we use the default libvirt network for attaching the edpm node and do a static dhcp reservations for crc and the edpm vms. this sometimes result in issues where when you re-create the VM and the new VM with a new mac gets a different IP address even if there is a static reservation. seems that dnsmasq still has a lease for the old one. the proper way could be like in this script to destroy and recreate the network and recreate it when the env get re-created, which we probably don't want to do with the default libvirt network.

Reply:
Sure, I'll move this script out of the bmaas subdirectory and do some slight modifications to make it more generic.

Reply:
@stuggi

I removed the "crc-bmaas" mentions in the script, moved it up to the /scripts directory.
Changed the default network name to crc-net-iso, and used 172.16.0.1/24 as the IP address.
I think that's all that is needed for generic use?

Reply:
yes, thanks! lgtm